sky/packages/sky/lib/src/gestures/events.dart

```diff
@@ -0,0 +1,114 @@
+import 'dart:ui' as ui;
+
+/// Dart-layer version of ui.Event
```

Comment:
I'd try documenting this without reference to ui.Event, since that class' days are numbered.